pam_authz/pam/README.md

```diff
@@ -0,0 +1,64 @@
+# Styra's PAM authorization module
+
+This implements a PAM module that integrates with Open Policy Agent (OPA)
+to make authorization decisions. When invoked, the module queries OPA
+and if no errors are returned, then the action is allowed. This allows
+administrators to implement fine-grained access control. For
+example, one can use this module to limit usage of `sudo` to particular
+times of day.
+
+This module is not intended to perform authentication. One can use standard
+linux accounts for that or a user database such as LDAP.
+
+
+## Usage:
+
+This repo includes a detailed [example usage of the pam_authz module](../docker/README.md),
```

Comment:
You want the docker files and docs moved to the OPA repo? Or just the docs moved to the OPA repo? Certainly we should have all examples in one place, but doesn't make sense to me to separate the docs from the code or to put the image building code into the OPA repo, so I'm not sure what you're proposing here